src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -49,6 +49,25 @@ def test_self_calibrate(multi_instrument_session):
     multi_instrument_session.self_calibrate()
 
 
+def test_channels_rep_cap(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.vil = 1
+    ch_0_63 = multi_instrument_session.get_channel_names(indices=[0, 63])
+    multi_instrument_session.channels[ch_0_63].vil = 2
+    assert multi_instrument_session.pins[ch_0_63].vil == pytest.approx(2, abs=1e-3)
+    ch_1 = multi_instrument_session.get_channel_names(indices=1)
+    assert multi_instrument_session.pins[ch_1].vil == pytest.approx(1, abs=1e-3)
+
+
+def test_sites_rep_cap(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.sites[0, 1].disable_sites()
+    assert not multi_instrument_session.sites[0].is_site_enabled()
+    assert not multi_instrument_session.sites[1].is_site_enabled()
```

Comment:
assert they are enabled before disabling

Reply:
you can merge after fixing.

Reply:
Fixed